pkgs/tools/system/howdy/default.nix

```diff
@@ -0,0 +1,55 @@
+{ lib
+, stdenv
+, fetchurl
+, dpkg
+, makeWrapper
+, python3
+, installShellFiles
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "howdy";
+  version = "2.6.1";
+
+  src = fetchurl {
+    url = "https://github.com/boltgolt/howdy/releases/download/v${finalAttrs.version}/howdy_${finalAttrs.version}.deb";
+    hash = "sha256-wQQynYx/VFCLMXrH+5eC6uQQQYe6xlfuioCu7rQGyE8=";
+  };
+
+  nativeBuildInputs = [ dpkg makeWrapper python3 installShellFiles ];
+
+  installPhase = ''
+    runHook preInstall
+
+    dpkg -x $src $out
```

Comment:
Hasn't the default `unpackPhase` already unpacked the source file?

Reply:
now explicitly calling `unpackPhase` in https://github.com/NixOS/nixpkgs/pull/254359/commits/950df87dbb3f3e04873be8ca7109f7d40fd037c7

Reply:
I wonder if an equivalent can be done in `postInstall`, without explicitly defining `unpackPhase` and calling `dpkg -x`. I'll have a look when I get home.